presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ITERATIVE_OPTIMIZER_TIMEOUT = "iterative_optimizer_timeout";
```

Comment:
@arhimondr still relevant, update the commit message for the planner part of merge